openapi/components/schemas/GatewayAccountConfig/Payretailers.yaml

```
@@ -0,0 +1,20 @@
+description: Payretailers gateway configuration.
+allOf:
+  - $ref: ../GatewayAccount.yaml
+  - type: object
+    required:
+      - credentials
+    properties:
+      credentials:
+        type: object
+        description: Payretailers credentials object.
+        properties:
+          shopId:
+            type: string
+          secretKey:
+            type: string
```

nitpick: We should add descriptions for all properties
I'm not totally sure if I have changed the meaning here

```suggestion
            type: string
            description: Secret key value of the PayRetailers API.
```